jekyll-import.gemspec

```diff
@@ -25,29 +25,30 @@ Gem::Specification.new do |s|
   # runtime dependencies
   s.add_runtime_dependency("jekyll", ">= 3.7", "< 5.0")
   s.add_runtime_dependency("nokogiri", "~> 1.0")
-  s.add_runtime_dependency("reverse_markdown", "~> 1.0")
+  s.add_runtime_dependency("reverse_markdown", "~> 2.1")
 
   # development dependencies
   s.add_development_dependency("bundler")
-  s.add_development_dependency("rake", "~> 12.0")
+  s.add_development_dependency("rake", "~> 13.0")
   s.add_development_dependency("rdoc", "~> 6.3.0")
 
   # test dependencies:
   s.add_development_dependency("redgreen", "~> 1.2")
   s.add_development_dependency("rr", "~> 1.0")
   s.add_development_dependency("rubocop-jekyll", "~> 0.11.0")
-  s.add_development_dependency("shoulda", "~> 3.5")
+  s.add_development_dependency("shoulda", "~> 4.0")
   s.add_development_dependency("simplecov", "~> 0.7")
   s.add_development_dependency("simplecov-gem-adapter", "~> 1.0")
 
   # migrator dependencies:
-  s.add_development_dependency("behance", "~> 0.3")
+  # s.add_development_dependency("behance", "~> 0.3") # uses outdated dependencies
```

Comment:
Shall we move this to the Gemfile and add a Ruby version check or just wait until someone comes around and tries to add tests for the Behance importer?

Reply:
We shall wait for someone to add tests for the Behance importer or at minimum report a bug or some other issue with the Behance importer..
I'm not keen on moving dev_deps to Gemfile (for this project) because the Rubygems dev_deps listing may serve as secondary documentation for users.

Reply:
Ok. Commenting this out removes it from the dev deps anyway, so I'm not sure it's of much use here.